src/xrc/xh_aui.cpp

```diff
@@ -193,6 +193,7 @@ wxObject *wxAuiXmlHandler::DoCreateResource()
             if ( HasParam(wxS("floatable")) )       paneInfo.Floatable( GetBool(wxS("floatable")) );
 // Sizes
             if ( HasParam(wxS("floating_size")) )   paneInfo.FloatingSize( GetSize(wxS("floating_size")) );
+            if ( HasParam(wxS("floating_client_size")) )   paneInfo.FloatingClientSize( GetSize(wxS("floating_client_size")) );
```

Comment:
Could you please also add this to `docs/doxygen/overviews/xrc_format.h` and `misc/schema/xrc_schema.rnc`, near the existing `floating_size`?

Reply:
Done.  See 8c9c2d608514df13b13a486553d100f492762d34.